libcontainer/dmz/Makefile

```diff
@@ -0,0 +1,6 @@
+# Get CC values for cross-compilation.
+include ../../cc_platform.mk
+
+runc-dmz: _dmz.c
+	$(CC) $(CFLAGS) -static -o $@ $^
```

Comment:
We can use the kernel's nolibc can be easily used and produce a very small binary. It seems only ppc is not supported, so we can fallback to stdlib in that case. We can just vendor it and no need to have anything installed on the distro building it.

A quick test shows that the binary can be 8K:
```
$ make
gcc  -static -fno-asynchronous-unwind-tables -fno-ident -s -Os -nostdlib -static -include nolibc/nolibc.h -o runc-dmz _dmz.c
strip -gs runc-dmz
$ du -h runc-dmz 
8.0K	runc-dmz
```

Reply:
We can do that in a separate PR -- I suspect we will need to make sure that it will not be annoying for distributions (and Docker) to package this.

Reply:
Ah, it's a header-only thing, that's right (it's also LGPL-MIT dual-licensed). Either way, let's do this in a separate PR.

Reply:
Yes, and ppc has just been added in 6.6, so it really looks very nice for this